src/kalite-app.py

```diff
@@ -42,5 +42,7 @@ def request_portal(url):
 sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
 connection = sock.connect_ex(('127.0.0.1', port))
 
-request_portal('http://localhost:8008')
+# We use a custom URI that will be handled by chromium-browser-appmode
+# so that chromium-browser gets launched with --class and -app set.
+request_portal('webapp://org.learningequality.KALite@http://localhost:8008')
```

Comment:
Yeah, I'm interested in catching the error from the portal when it can't find a suitable app to handle the webapp scheme.